datastore/google/cloud/datastore/key.py

```diff
@@ -324,7 +326,10 @@ def from_legacy_urlsafe(cls, urlsafe):
         This is intended to work with the "legacy" representation of a
         datastore "Key" used within Google App Engine (a so-called
         "Reference"). This assumes that ``urlsafe`` was created within an App
-        Engine app via something like ``ndb.Key(...).urlsafe()``.
+        Engine app via something like ``ndb.Key(...).urlsafe()``.Notice that 
```

Comment:
See my above about padding and also you are missing a space between the previous sentence and the start of your "Notice..."